.github/workflows/traffic-ops-ort-tests.yml

```diff
@@ -0,0 +1,153 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Traffic Ops ORT integration tests
+
+on: 
+  push:
+  workflow_dispatch:
+    paths:
+      - .github/workflows/traffic-ops-ort.yml
+      - go.mod
+      - go.sum
+      - GO_VERSION
+      - traffic_ops/*client/**.go
+      - traffic_ops/traffic_ops_golang/**.go
+      - traffic_ops_ort/**.go
+      - vendor/**.go
+      - vendor/modules.txt
+  create:
+  pull_request:
+    paths:
+      - .github/workflows/traffic-ops-ort.yml
+      - go.mod
+      - go.sum
+      - GO_VERSION
+      - traffic_ops/*client/**.go
+      - traffic_ops/testing/api/**.go
```

Comment:
this action will still run on changes to the Go client/TO API integration tests on PRs

Reply:
@ocket8888 that's intended, the client stuff is used in ORT.

Reply:
Sure the TO clients are used in ORT. I think @ocket8888 is talking specifically about

```yaml
    - traffic_ops/testing/api/**.go
```
because no package within `github.com/apache/trafficcontrol/traffic_ops/testing/api` is used by ORT as far as I can tell.

Reply:
@zrhoffman my mistake.  I was looking at the wrong path, it's been a long day :).  I just pushed the change.